tests/suites/caasadmission/task.sh

```diff
@@ -6,20 +6,20 @@ test_caasadmission() {
 
 	set_verbosity
 
-  case "${BOOTSTRAP_PROVIDER:-}" in
-    "k8s")
-      echo "==> Checking for dependencies"
-      check_dependencies petname
```

Comment:
I assume the rest of these are just spaces vs tabs? As long as we end up consistent

Reply:
yes, static analysis caught them, ran shfmt on the files.